skopt/sample.py

```diff
@@ -0,0 +1,79 @@
+from collections import Mapping
+from numbers import Integral
+from numbers import Real
+
+from scipy.stats._distn_infrastructure import rv_frozen
+from scipy.stats.distributions import randint
+from scipy.stats.distributions import uniform
+
+from sklearn.utils import check_random_state
+from sklearn.utils.fixes import sp_version
+
+
+class Categorical:
+    def __init__(self, *args):
+        self.values = args
+
+    def rvs(self, random_state=None):
+        rng = check_random_state(random_state)
+        return self.values[rng.randint(len(self.values))]
+
+    @property
+    def low(self):
```

Comment:
Actually, we should briefly discuss if it makes sense to "order" categoricals. What is the order of: banana, apple, orange, chocolate?

What is the use case of extracting the bounds now that we can sample directly? Maybe we can get rid of `low` and `high`. This would make it simple to accept arbitrary `scipy.stats` as well.